gcc/rust/hir/rust-ast-lower.cc

```diff
@@ -495,6 +495,18 @@ void
 ASTLowerPathInExpression::visit (AST::PathInExpression &expr)
 {
   std::vector<HIR::PathExprSegment> path_segments;
+  auto crate_num = mappings.get_current_crate ();
+  Analysis::NodeMapping mapping (crate_num, expr.get_node_id (),
+				 mappings.get_next_hir_id (crate_num),
+				 UNKNOWN_LOCAL_DEFID);
+  if (expr.get_path_kind () == AST::Path::Kind::LangItem)
+    {
+      translated
+	= new HIR::PathInExpression (mapping, std::move (path_segments),
+				     expr.get_lang_item (), expr.get_locus (),
```

Comment:
so we'd get rid of the segments here for example

```suggestion
	= new HIR::PathInExpression (mappin, expr.get_lang_item (), expr.get_locus (),
```

Reply:
Fixed